templates/home/header.html.twig

```diff
@@ -1,9 +1,11 @@
 <header class="container bg-white">
     <nav class="navbar navbar-expand-lg navbar-light">
         <a class="navbar-brand" href="/">
-            <img src="{{asset('build/images/logo.svg')}}" width="160" height="90" class="d-inline-block align-top" alt="Lita logo">
+            <img src="{{ asset('build/images/logo.svg') }}" width="160" height="90" class="d-inline-block align-top"
+                 alt="Lita logo">
```

Comment:
If you're moving tag attributes to new lines, then all attributes should be moved to the new lines.